widgets/editor_widgets.py

```diff
@@ -46,20 +48,18 @@ def handle(*args, **kwargs):
             return func(*args, **kwargs)
         except Exception as e:
             traceback.print_exc()
-            open_error_dialog(str(e), None)
+
+            parent = QtWidgets.QApplication.instance().editor_gui
+            open_error_dialog(str(e), parent)
     return handle
 
 
-def open_error_dialog(errormsg, self):
-    errorbox = QtWidgets.QMessageBox()
-    errorbox.critical(self, "Error", errormsg)
```

Comment:
Note that [`QtWidgets.QMessageBox.critical()`](https://doc.qt.io/qt-6/qmessagebox.html#critical) is a static method; it does not use an instance. The creation of the `errorbox` object is redundant, and the `setFixedSize()` call had no effect.

(Same goes for the function below.)